datacube/scripts/dataset.py

```diff
@@ -444,7 +444,7 @@ def search_cmd(index, limit, f, expressions):
     )
 
 
-def _get_derived_set(index, id_):
+def _get_derived_set(index: Index, id_: str) -> Set[Dataset]:
```

Comment:
Should this be `id_: Union[str, UUIDr]`, elsewhere I used `LaxUUID = Union[str, UUID]` up-top for that sort of situation.

Reply:
Yeah, probably should be. I looked at local usages and all were giving strings so I went with simple. 

... but that typedef would be simple and more correct.